aten/src/ATen/native/quantized/cpu/serialization_versions.h

```diff
@@ -0,0 +1,164 @@
+#pragma once
+
+#include <ATen/ATen.h>
+#include <ATen/core/List.h>
+
+#include <ATen/native/quantized/cpu/fbgemm_utils.h>
+#include <ATen/native/quantized/cpu/qnnpack_utils.h>
+
+#include <tuple>
+
+using SerializationType = std::tuple<
+  std::string /*name of the op that is serialized*/,
```

Comment:
why do we serialize this?

Reply:
I assumed this would be if `SerializationType` is used for other ops in the future

Reply:
when this is used for other ops, why do we need to write down the name here?(just went through similar process with @supriyar 's PR actually). There are two questions to ask here:
(1). is it necessary to distinguish between different ops? if not, we can just remove it
(2). if so, can we put this as a template argument of the type? e.g. ConvPackedParams<1> ConvPackedParams<2> for conv1d and conv2d
(3). even we need to distinguish the op name and can't put it in template, I would consider serializing integer(enum) values better than serializing strings since this is replicated for all instances.